SeaPkg/Tools/GenSeaArtifacts/GenSeaArtifacts.py

```diff
@@ -231,6 +233,57 @@ def generate_sea_manifest(stm_bin: Path, output_path: Path, config: dict, worksp
         
         return 0
 
+    @staticmethod
+    def validate_rule_coverage(percent: int, aux_path: Path):

```

Comment:
Was `percent` meant to be an `int` instead of a `float`?

Reply:
Yes I did int purposefully, but I'd say you are right, a float is better. I'll update.

Reply:
Updated.